spark/src/main/java/io/jaegertracing/analytics/spark/SparkKinesisRunner.java

```diff
@@ -63,14 +65,27 @@
      * @throws IOException
      */
     public static void main(String[] args) throws InterruptedException, IOException {
+         PropertyConfigurator.configure("/data/log4j.properties"); // can make it configurable
```

Comment:
Why do we have to do like this? Log4j2 will look for config file specified in a pre defined env var by default. We can set that env var to our log4j2.xml or log4j2.yml file when we run this jar. 

``` groovy 
<dependency>
    <groupId>org.apache.logging.log4j</groupId>
    <artifactId>log4j-api</artifactId>
    <version>2.14.1</version>
</dependency>
<dependency>
    <groupId>org.apache.logging.log4j</groupId>
    <artifactId>log4j-core</artifactId>
    <version>2.14.1</version>
</dependency>
```
https://logging.apache.org/log4j/2.x/manual/configuration.html